### PR TITLE
chore(overview): translate docs/index.md

### PR DESCRIPTION
**Pull request by takam5f2**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,52 +1,71 @@
...
+The fork of rclcpp which has CARET-dedicated tracepoints is delivered.
+You have to build CARET and your application if you want to use.
 
-測定を行うとトレース結果が得られ、この結果を元に、解析を行います。  
-解析前には、レイテンシの定義や測定対象のパスの定義を記述したアーキテクチャファイルが必要になります。
+If you run your application with CARET, you will get a set of trace data. If you analyze the data set, you have to create an architecture file in which node latency and target path are defined.
 
-解析用のパッケージ、トレース結果、アーキテクチャファイルが揃い、性能の評価や解析が可能になります。  
-一部 CLI での可視化も対応していますが、主に jupyter 上での評価・解析することを想定しています。
+You will visualize trace data with the architecture file and `CARET_trace` package, including API for data analysis.
```
Comment (hsgwa) [inline docs/index.md:43]: `CARET_trace` -> `CARET_analyze`

File: docs/index.md [hunk 2]
```diff
@@ -1,52 +1,71 @@
...
-フォークしてトレースポイントを追加した ROS を必要とします。  
-現在はソースコードでの提供のみになっているため、これらパッケージのビルドが必要です。
+CARET is served as only source code, but not as `apt` package, so far.  
+CARET hooks dedicated functions to those defined in dynamic library in order to add tracepoints.  
+The fork of rclcpp which has CARET-dedicated tracepoints is delivered.
+You have to build CARET and your application if you want to use.
 
-測定を行うとトレース結果が得られ、この結果を元に、解析を行います。  
-解析前には、レイテンシの定義や測定対象のパスの定義を記述したアーキテクチャファイルが必要になります。
+If you run your application with CARET, you will get a set of trace data. If you analyze the data set, you have to create an architecture file in which node latency and target path are defined.
```
Comment (hsgwa) [inline docs/index.md:41]: Personally, I find the following text easier to read.
> you have to create an architecture file which defines node latency and target path.

**Review by hsgwa — CHANGES_REQUESTED** [review]

File: docs/index.md (hunk 2 above)
Comment (takam5f2) [reply docs/index.md]: How about updated text? I think that "user has to define node latency and target path", but a file does not define.

File: docs/index.md (hunk 1 above)
Comment (takam5f2) [reply docs/index.md]: Fixed in https://github.com/tier4/CARET_doc/pull/43/commits/9045f9c8d6b9cd73ac347f17bf6d5944c807ca03

File: docs/index.md (hunk 2 above)
Comment (hsgwa) [reply docs/index.md]: make sense.

**Review by hsgwa — APPROVED** [review]

**Pull request merged**